src/development/going-live.md

```diff
@@ -11,13 +11,13 @@ This page assumes you already have the following:
 3. If your domain is currently active elsewhere, the Time-To-Live (TTL) on your domain is set to the lowest possible value in order to minimize transition time.
 4. You have the auto-generated domain for your master branch.  This is the domain you see in the Location bar after selecting "Access site" in the UI.  You can also retrieve this value from the command line by running `platform environment:url` to see a list of all URLs that Platform.sh will serve for the current environment.  Write this down.
 5. Optional: If you want to guarantee that you have access to  your master environment before the domain name has switched over, use `ping` or any similar tool to determine the IP address of the master environment.  The IP address is not guaranteed stable but is unlikely to change during the course of the go-live process.
-6. Optional: If you want to use an SSL certificate to encrypt your production site (you do), you can obtain one from any number of 3rd party SSL issuers.  Platform.sh does not charge anything to use SSL in production, although at this time we do not issue our own certificates.
+6. Optional: If you want to use a 3rd party SSL certificate to encrypt your production site, you can obtain one from any number of 3rd party SSL issuers.  Platform.sh automatically provides SSL certificates for all sites issued by LetsEncrypt.
```

Comment:
Let's Encrypt

Also, looks like the "no charge" message is lost for a custom cert.